python/lsst/pex/config/config.py

```diff
@@ -237,23 +237,26 @@ def save(self, outfile, instance):
         """
         value = self.__get__(instance)
         fullname = _joinNamePath(instance._name, self.name)
+
+        # write full documentation string as comment lines (i.e. first character is #)
+        doc = "# " + re.sub(r'\n', r'\n# ', str(self.doc))
         if isinstance(value, float) and (math.isinf(value) or math.isnan(value)):
             # non-finite numbers need special care
-            print >> outfile, "%s=float('%r')"%(fullname, value)
+            print >> outfile, "%s\n%s=float('%r')\n"%(doc, fullname, value)
```

Comment:
But I do think that using print in a config file is an abuse of the system.  Yes it's a Python script, but you're not supposed to use it to do non-config things.